doc/making-a-visualizer.md

```diff
@@ -140,42 +135,35 @@ Sequence information is found in `this.seq`. This is a `SequenceInterface`
 object, so it will always have a method `getElement(n)` that returns the `n`th
 element in the sequence.
 
-#### Stop and start animation
+You have to implement this method, even if it does nothing. Your visualizer
+can't be loaded into Numberscope without it.
+
+#### :bulb: Stop and start animation _(often used)_
 
 If your visualization is a static picture, call the sketch's `noLoop()`
 function when you're done drawing. This stops the animation loop, so you don't
 waste time re-drawing the same picture dozens of times per second. If the
 visualization is only static temporarily, you can call `loop()` to re-start
 the animation loop whenever you need to.
 
-One interesting animation design pattern, used in visualizers like Mod Fill,
-is to iterate through one sequence element per frame, refining the
-visualization each time. Once enough elements have been processed, the
-visualizer calls `noLoop()`, leaving a static, finished picture. This makes
-the visualizer seem to build itself up over a a fraction of a second.
-
 -   **p5 Visualizer Template:** Only loop while animating the white flash that
     indicates an index change.
 -   **Differences:** Draw the whole visualization in one frame, and then call
     `noLoop()` to stop the animation loop.
 
-#### Respond to user interactions
+#### :bulb: Respond to user interactions _(often used)_
 
 Each time the user interacts with your visualization, an
 [event handling](https://p5js.org/reference/#group-Events) function like
 `keyPressed()` or `mouseClicked()` will be called, giving you a chance to
 respond. There are handlers for a wide variety of input events.
```

Comment:
maybe put a link to the list of such?  there's one such list in the code somewhere that Glen put in when he was implementing it.

Reply:
@katestange Aaron does (now) have an external link to the relevant section of the p5 reference page. Is that enough, or would you like an explicit listing here of the ones we've supported in P5Visualizer.ts? If you think that explicit list should be here, should it just be a list of names interpolated into the text right here in place, or a linked internal doc page giving each of the methods a P5Visualizer can define and maybe a one-sentence description for each of the circumstances under which it will be called? Thanks for clarifying, any of the above should be feasible (and Aaron, if you need a hand extracting rather than reiterating info like this using mkdocs, let me know).

Reply:
Since Kate is not likely to be responding soon, I will say that I think in the current context/timeframe just the external link is enough for now. I will leave this conversation unresolved though, in case the review process does go on long enough that Kate takes another look before it is done. Then we can resolve it just before merging if not; and then if later it seems like an explicit list either here or on another doc page would be good, we can file an issue for that.